Add Tanium initialization file and DHS PKI bundle to installation procedure

jsf9k opened the pull request:
## 🗣 Description ##

This pull requests adds to this role:
* A Tanium initialization file that is installed
* The installation and trusting (at the OS level) of the DHS/Treasury PKI bundle

## 💭 Motivation and context ##

The initialization file contains a registration secret that was found to be necessary when debugging with the VENOM team.

Without these certificates the Tanium clients in the COOL could not communicate with the VENOM Tanium server at all.  With these certificates the SYN packets could be seen travelling from the COOL, through the site-to-site VPN tunnel, and reaching the VENOM Tanium server; however, no ACK packets were received by the COOL systems.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

Without the registration secret in the initialization it was not possible to communicate at all with the Tanium server.  With the initialization file we could see the communication begin.

Without the DHS PKI certificates the Tanium clients in the COOL could not communicate with the VENOM Tanium server at all. With these certificates the SYN packets could be seen travelling from the COOL, through the site-to-site VPN tunnel, and reaching the VENOM Tanium server; however, no ACK packets were received by the COOL systems.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

jsf9k committed:
Add the Tanium initialization file to the objects that can be gotten from the S3 bucket

jsf9k committed:
Add the Tanium init file to the installation procedure

jsf9k committed:
Start the Tanium client service in the converge playbook

This allows us to later test in molecule that a registration secret was set via the initialization file.

dav3r commented on defaults/main.yml:
```diff
@@ -1,4 +1,9 @@
 ---
+# A tanium initialization file containing Tanium configuration
```
Sorry to nitpick....
```suggestion
# A Tanium initialization file containing Tanium configuration
```

dav3r reviewed (approved):
It's Tantastic! 🍊

jsf9k committed:
Capitalize Tanium for consistency

Co-authored-by: dav3r <david.redmin@trio.dhs.gov>

jsf9k committed:
Add cisagov/ansible-role-dhs-certificates as a dependency

These certificates are also necessary to communicate with the VENOM Tanium server.

jsf9k commented:
I'm about to alter the history of this repository, so to be safe I will go ahead and merge this PR.
